libvirt/image/Makefile

```diff
@@ -120,6 +122,19 @@ $(PAUSE_SRC): $(SKOPEO_SRC)/bin/skopeo
 $(PAUSE): | $(PAUSE_SRC) $(UMOCI_SRC)/umoci
 	$(UMOCI_SRC)/umoci unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
 
+docker-build:
+	docker build -t caa-libvirt-image-builder \
+	-f Dockerfile_ubuntu.builder .
+	docker run --rm \
+		-v $(PROJECT_DIR):/src/cloud-api-adaptor \
+		-v $(abspath $(PROJECT_DIR)/../kata-containers):/src/kata-containers \
+		-e UID=$(shell id -u) \
+		-e GID=$(shell id -g) \
+		caa-libvirt-image-builder \
+		bash -c 'trap "chown -R -f "$$UID:$$GID" output staticlib $(SKOPEO_SRC) \
+		$(UMOCI_SRC) $(PAUSE_SRC) $(FILES_DIR)/$(PAUSE_BUNDLE) $(FILES_DIR)/usr \
+		../../agent-protocol-forwarder || true" EXIT; \
```

Comment:
can you explain this line?

Reply:
Magic my friend, magic...

Those files are created with root permission and I don't like it. For example, afterwards when I run `make clean` I get permission denied. That line ensure that `chown` will run on bash-exit and then will change the files ownership to the user that called docker. I know that there are other ways to address that issue, for example, create an user inside the container with same id; but I thought it would be too complicated. If you have a easier way to solve it then please let me know.